src/app/qt/facefilter/AndroidManifest.xml

```diff
@@ -70,7 +70,7 @@
       <!-- Background running -->
     </activity>
   </application>
-  <uses-sdk android:minSdkVersion="9" android:targetSdkVersion="@CMAKE_SYSTEM_VERSION@"/>
+  <uses-sdk android:minSdkVersion="19" android:targetSdkVersion="@CMAKE_SYSTEM_VERSION@"/>
```

Comment:
For `minSdkVersion="19"` we can assume OpenGL ES 3.0.

Actually, I guess that was added in `android-18`.  This will probably be critical for reasonable performance on Android devices since OpenGL ES 3.0 is needed for PBO use and the `dlopen` + `GraphicBuffer` platform optimizations are blocked in the latest Android release.
https://developer.android.com/guide/topics/graphics/opengl.html

There are only a handful of devices that are < KitKat `android-19`.  I think we might be able to drop it to `android-18`, but the supported CI toolchains are currently all `android-19.

https://developer.android.com/about/dashboards/index.html#Platform